site/_docs/reference.md

```diff
@@ -2739,6 +2739,9 @@ BigQuery's type system uses confusingly different names for types and functions:
 | b | FORMAT_TIMESTAMP(string timestamp)             | Formats *timestamp* according to the specified format *string*
 | s | GETBIT(value, position)                        | Equivalent to `BIT_GET(value, position)`
 | b o | GREATEST(expr [, expr ]*)                    | Returns the greatest of the expressions
+| h s | HEX(binary)                                  | Converts *binary* into a hexadecimal varchar. For example, hex(x'6162') returns '6162'
+| h s | HEX(bigint)                                  | Converts *bigint* into a shortened hexadecimal varchar. For example, hex('17') returns '11'
```

Comment:
what does it mean "shortened"?

Reply:
The leading zeros will be removed to keep consistent with the behaviour of Hive and Spark.

Reply:
It would be better to change example to `hex('10') returns 'a'`.

Reply:
Updated.